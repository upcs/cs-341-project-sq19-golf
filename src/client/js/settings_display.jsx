import React, { Component } from 'react';
import {Link} from 'react-router-dom';
import '../css/styles.css';

export class SettingsContainer extends Component {
  render() {
    return (
      <div>
      </div>
    )
  }
}
