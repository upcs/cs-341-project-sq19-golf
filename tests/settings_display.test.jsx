import React from 'react';
import { shallow } from 'enzyme';

import {SettingsContainer} from '../src/client/js/settings_display.jsx';

describe('AvailabilityContainer', () => {
  test('Should render correctly', () => {
    const settingsContainer = shallow(<SettingsContainer/>);
    expect(settingsContainer).toMatchSnapshot();
  });
});
