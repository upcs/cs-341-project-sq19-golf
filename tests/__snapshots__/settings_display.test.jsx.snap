// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AvailabilityContainer Should render correctly 1`] = `
<section
  id="main"
>
  <button
    onClick={[Function]}
  >
    About
  </button>
  <Link
    replace={false}
    to="/"
  >
    <button
      id="quit"
      type="button"
    >
      Quit
    </button>
  </Link>
</section>
`;
